examples/particles.ipynb

```diff
@@ -0,0 +1,272 @@
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": null,
+   "metadata": {},
+   "outputs": [],
+   "source": [
+    "from expertsystem.state import particle\n",
+    "from expertsystem.ui.system_control import load_default_particle_list\n",
+    "\n",
+    "print(\"Before loading:\", len(particle.DATABASE))\n",
+    "load_default_particle_list()\n",
+    "print(\"After loading:\", len(particle.DATABASE))"
+   ]
+  },
+  {
+   "cell_type": "raw",
+   "metadata": {
+    "raw_mimetype": "text/restructuredtext"
+   },
+   "source": [
+    "The functions in the :mod:`.particle` module can be used to add or modify particle definitions in the particle database. There are a few ways to add or overwrite particle definitions."
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "## Adding custom particle definitions through Python"
+   ]
+  },
+  {
+   "cell_type": "raw",
+   "metadata": {
+    "raw_mimetype": "text/restructuredtext"
+   },
+   "source": [
+    "A quick way to modify or overwrite particles, is through your Python script of notebook. It's quite easy to modify one particle from the particle database, as it's simply a `dict`:"
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": null,
+   "metadata": {},
+   "outputs": [],
+   "source": [
+    "omega = particle.DATABASE[\"omega(782)\"]\n",
+    "omega"
+   ]
+  },
+  {
+   "cell_type": "raw",
+   "metadata": {
+    "raw_mimetype": "text/restructuredtext"
+   },
+   "source": [
+    "A useful (even if rather hackable) modification is for instance the width. In the `expertsystem`, the width is used for the mass conservation check, while in `tensorwaves`, the width is important when generating a toy MC sample (see :ref:`tensorwaves:usage/2_generate_data:2.3 Generate intensity-based sample`):"
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": null,
+   "metadata": {},
+   "outputs": [],
+   "source": [
+    "omega[\"DecayInfo\"][\"Parameter\"][0][\"Value\"] = 0.001"
+   ]
+  },
+  {
+   "cell_type": "raw",
+   "metadata": {
+    "raw_mimetype": "text/restructuredtext"
+   },
+   "source": [
+    "More relevant for your research is adding particles. Here it is useful to work with an existing particle as template. Let's say we want to study :math:`e^+e^-` collisions of several energies:"
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": null,
+   "metadata": {},
+   "outputs": [],
+   "source": [
+    "energies_mev = [4180, 4220, 4420, 4600]\n",
+    "for counter, energy_mev in enumerate(energies_mev, 1):\n",
+    "    energy_gev = energy_mev / 1e3\n",
+    "    new_particle = particle.get_particle_copy_by_name(\"EpEm (4230 MeV)\")\n",
+    "    new_particle[\"Name\"] = f\"EpEm ({energy_mev} MeV)\"\n",
+    "    new_particle[\"Parameter\"][\"Value\"] = energy_gev  # set the mass\n",
+    "    new_particle[\"Pid\"] = int(new_particle[\"Pid\"]) + counter\n",
```

Comment:
Is there "PID" instead of "Pid"?

Reply:
Well spotted!
Yeah so this is the problem with the internal `dict` structure, which is based on the `particle_list.xml` (XML). There, it is `"Pid"` (see e.g. [here](https://github.com/ComPWA/expertsystem/blob/87f143dec7a419bd1d3a592ca150ce0e37a1810c/expertsystem/particle_list.xml#L7)), while in YAML (which has a much better structure, used for `tensorwaves`), it is `"PID"`.